packages/web-react/src/utils/string.ts

```diff
@@ -1,6 +1,48 @@
+// Reusable regex transform function
+const kebabToCamelTransform = (str: string): string => str.replace(/-([a-z])/g, (g) => g[1].toUpperCase());
+
 /**
- * Converts a kebab-case string to camelCase
+ * Converts a kebab-case string to camelCase.
  *
- * @param str
+ * @param {string} str - The kebab-case string to be converted.
+ * @returns {string} The camelCase version of the input string.
  */
-export const kebabCaseToCamelCase = (str: string): string => str.replace(/-([a-z])/g, (g) => g[1].toUpperCase());
+export const kebabCaseToCamelCase = (str: string): string => kebabToCamelTransform(str);
```

Comment:
**suggestion (if-minor):** Why is here this function? It is only an alias to the original function.